source/cpp/source/DefaultCommandHandler.cpp

```diff
@@ -512,6 +512,17 @@ juce::String getAccessibilityHelpText (juce::Component & component)
                : component.getHelpText ();
 }
 
+juce::String getAccessibilityParent (juce::Component & component)
```

Comment:
Could this be a `const juce::Component &`?

Reply:
No, because accessing the accessibility handler is non const :(